Icon/Icons.php

```diff
@@ -0,0 +1,1202 @@
+            self::FA_TRASH_O => 'fa-trash-o',
+            self::FA_TREE => 'fa-tree',
+            self::FA_TRELLO => 'fa-trello',
+            self::FA_TRIPADVISOR => 'fa-tripadvisor',
+            self::FA_TROPHY => 'fa-trophy',
+            self::FA_TRUCK => 'fa-truck',
+            self::FA_TTY => 'fa-tty',
+            self::FA_TUMBLR => 'fa-tumblr',
+            self::FA_TUMBLR_SQUARE => 'fa-tumblr-square',
+            self::FA_TURKISH_LIRA => 'fa-turkish-lira',
+            self::FA_TV => 'fa-tv',
+            self::FA_TWITCH => 'fa-twitch',
+            self::FA_TWITTER => 'fa-twitter',
+            self::FA_TWITTER_SQUARE => 'fa-twitter-square',
+            self::FA_UMBRELLA => 'fa-umbrella',
+            self::FA_UNDERLINE => 'fa-underline',
+            self::FA_UNDO => 'fa-undo',
+            self::FA_UNLINK => 'fa-unlink',
+            self::FA_UNLOCK => 'fa-unlock',
+            self::FA_UNLOCK_ALT => 'fa-unlock-alt',
+            self::FA_UNSORTED => 'fa-unsorted',
+            self::FA_UPLOAD => 'fa-upload',
+            self::FA_USD => 'fa-usd',
+            self::FA_USER => 'fa-user',
+            self::FA_USER_MD => 'fa-user-md',
+            self::FA_USER_PLUS => 'fa-user-plus',
+            self::FA_USER_SECRET => 'fa-user-secret',
+            self::FA_USER_TIMES => 'fa-user-times',
+            self::FA_USERS => 'fa-users',
+            self::FA_VENUS => 'fa-venus',
+            self::FA_VENUS_DOUBLE => 'fa-venus-double',
+            self::FA_VENUS_MARS => 'fa-venus-mars',
+            self::FA_VIACOIN => 'fa-viacoin',
+            self::FA_VIDEO_CAMERA => 'fa-video-camera',
+            self::FA_VIMEO => 'fa-vimeo',
+            self::FA_VIMEO_SQUARE => 'fa-vimeo-square',
+            self::FA_VINE => 'fa-vine',
+            self::FA_VK => 'fa-vk',
+            self::FA_VOLUME_DOWN => 'fa-volume-down',
+            self::FA_VOLUME_OFF => 'fa-volume-off',
+            self::FA_VOLUME_UP => 'fa-volume-up',
+            self::FA_WARNING => 'fa-warning',
+            self::FA_WEIBO => 'fa-weibo',
+            self::FA_WEIXIN => 'fa-weixin',
+            self::FA_WHATSAPP => 'fa-whatsapp',
+            self::FA_WHEELCHAIR => 'fa-wheelchair',
+            self::FA_WIFI => 'fa-wifi',
+            self::FA_WIKIPEDIA_W => 'fa-wikipedia-w',
+            self::FA_WINDOWS => 'fa-windows',
+            self::FA_WON => 'fa-won',
+            self::FA_WORDPRESS => 'fa-wordpress',
+            self::FA_WRENCH => 'fa-wrench',
+            self::FA_XING => 'fa-xing',
+            self::FA_XING_SQUARE => 'fa-xing-square',
+            self::FA_Y_COMBINATOR_SQUARE => 'fa-y-combinator-square',
+            self::FA_YAHOO => 'fa-yahoo',
+            self::FA_YC => 'fa-yc',
+            self::FA_YELP => 'fa-yelp',
+            self::FA_YEN => 'fa-yen',
+            self::FA_YOUTUBE => 'fa-youtube',
+            self::FA_YOUTUBE_PLAY => 'fa-youtube-play',
+            self::FA_YOUTUBE_SQUARE => 'fa-youtube-square',
+        );
+    }
+}
```

Comment:
:no_entry: This is a big no-no. :no_entry: 
This data should be loaded on demand from an external file, like [this one](https://github.com/FortAwesome/Font-Awesome/blob/master/src/icons.yml), which should be commited to [SonataCoreBundle](https://github.com/sonata-project/SonataCoreBundle/tree/3.x/Resources/public/vendor/components-font-awesome).